functions.php

```diff
@@ -133,3 +133,44 @@ function twentytwentyfive_pattern_categories() {
 endif;
 
 add_action( 'init', 'twentytwentyfive_pattern_categories' );
+
+add_action( 'init', 'twentytwentyfive_register_block_bindings', 9 );
```

Comment:
Is there any particular reason why you are setting the priority to 9?

We can move this one below the function to match what we did on the rest.

Reply:
Well, it is intended to run before _something_, but I can't for the life of me remember what.